requirements/requirements.txt

```diff
@@ -141,7 +141,7 @@ gitdb==4.0.2
     # via gitpython
 gitpython==3.1.7
     # via -r /awx_devel/requirements/requirements.in
-google-auth==1.11.3
+google-auth==1.35.0
```

Comment:
Kubernetes wants what kubernetes wants :+1:

Reply:
This package is actually at 2.3.3 now. https://pypi.org/project/google-auth/#history

That seemed like a risky bump, so I verified the latest release of 1.x allowed for pulling in the required version of `rsa` and used that.